config/config.go

```diff
@@ -10,7 +10,8 @@ import (
 
 // app meta
 const AppName = "prometheus2appoptics"
-var(
+
+var (
 	MajorVersion = 0
```

Comment:
warning: exported function PrintVersionAndExit should have comment or be unexported (golint)